src/components/modules/toolbar/inline.ts

```diff
@@ -362,20 +378,76 @@ export default class InlineToolbar extends Module {
     }
 
     const toolSettings = this.Editor.Tools.getToolSettings(currentBlock.name);
+    const inlineToolbarSettingsForTool = toolSettings && toolSettings[this.Editor.Tools.USER_SETTINGS.ENABLED_INLINE_TOOLS];
 
-    return toolSettings && toolSettings[this.Editor.Tools.USER_SETTINGS.ENABLED_INLINE_TOOLS];
+    /**
+     * Condition 1 : Returns true, if the inlineToolbar property of the block is an array or true;
+     * false otherwise.
+     */
+    const isInlineToolbarSettingsForToolProvided = inlineToolbarSettingsForTool == true || Array.isArray(inlineToolbarSettingsForTool);
+
+    /**
+     * Condition 2 : Returns true, if the inlineToolbar property of the block is not set,
+     * but the default inlineToolbar property is either an array or true.
+     */
+    const isDefaultInlineToolbarProvided = inlineToolbarSettingsForTool == undefined && this.config.inlineToolbar;
```

Comment:
```suggestion
    const isDefaultInlineToolbarProvided = inlineToolbarSettingsForTool === undefined && this.config.inlineToolbar;
```

Reply:
Also var name does not actually express its value

Reply:
`isDefaultInlineToolbarProvided` => Is an array provided for default inline toolbar property.
This is how i thought about it. Can u suggest a better name?